drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -257,7 +257,18 @@ def merge_candidates
 
     application_users_field :application_users
     def application_users(**args)
-      resolve_application_users(Hmis::User.all, **args)
+      raise 'access denied' unless current_user.can_audit_users? || current_user.can_impersonate_users?
+
+      resolve_application_users(Hmis::User.hmis_users, **args)
+    end
+
+    field :user, Types::Application::User, 'User lookup', null: true do
+      argument :id, ID, required: true
+    end
+    def user(id:)
+      raise 'access denied' unless id == current_user.id.to_s || current_user.can_audit_users? || current_user.can_impersonate_users?
+
+      Hmis::User.find_by(id: id)
```

Comment:
Should this be restricted to `Hmis::User.hmis_users` scope? Also might as well use the loader even for single-record query roots like this 🤷 

```suggestion
      load_ar_scope(scope: Hmis::User.hmis_users, id: id)
```

Reply:
I suppose it can be restricted down to hmis-only users. We should allow inactive because we're probably going to incorporate that into the UI at some point, so I'll leave those allowed